src/js/tech/ChromecastTech.js

```diff
@@ -238,19 +245,51 @@ module.exports = function(videojs) {
        * @see {@link http://docs.videojs.com/Tech.html#setCurrentTime}
        */
       setCurrentTime(time) {
-         var duration = this.duration();
+         if (this.scrubbing()) {
+            this._scrubbingTime = time;
+            return false;
+         }
+         const duration = this.duration();
 
          if (time > duration || !this._remotePlayer.canSeek) {
             return;
          }
-         // Seeking to any place within (approximately) 1 second of the end of the item
-         // causes the Video.js player to get stuck in a BUFFERING state. To work around
-         // this, we only allow seeking to within 1 second of the end of an item.
-         this._remotePlayer.currentTime = Math.min(duration - 1, time);
-         this._remotePlayerController.seek();
+
+         // We need to delay the actual seeking, because when you
+         // scrub, videojs does pause() -> setCurrentTime() -> play()
+         // and that triggers a weird bug where the chromecast stops sending
+         // time_changed events.
+         this._isSeeking = true;
+         setTimeout(() => {
```

Comment:
Unfortunately, this fix for  #103 is causing the seek bar to not be as responsive as before. Also, the delay does not allow for quick mouse clicks/taps to be registered on the seek bar.

However, we'd like to still get the `modifyLoadRequestFn` changes merged. Can the seek bar scrubbing changes be removed for now and placed into another PR? Thanks!

Reply:
Hi! Moved to #165 We can continue the discussion there.